odl/solvers/util/callback.py

```diff
@@ -325,12 +333,38 @@ def __init__(self, *args, **kwargs):
         self.kwargs = kwargs
         self.fig = kwargs.pop('fig', None)
         self.display_step = kwargs.pop('display_step', 1)
+        self.saveto = kwargs.pop('saveto', None)
+        self.suffix = kwargs.pop('suffix', 'png')
         self.iter = 0
 
+        if self.saveto is not None:
+            head, tail = os.path.split(self.saveto)
+            if not os.path.exists(head) or head == '':
+                raise ValueError('head of the `saveto` {} path not found'
```

Comment:
I have never heard the word head in this sense before, what do we mean more explicitly?